app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,22 @@
+.module {
+  width: auto;
```

Comment:
Properties should be ordered background, background-size, height, max-width, width